src/App.js

```diff
@@ -4,15 +4,57 @@ import './App.css';
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <h1>
-          Eric F. Olsen
-        </h1>
-        <h3>
-          Software Engineer
-        </h3>
-      </header>
+      <Header />
+      {/* <Story />
+      <Resume />
+      <Projects />
+      <Testimonials /> */}
+    </div>
+  );
+}
+
+function Header() {
+  return (
+    <header className="App-header">
+      <img src={logo} className="App-logo" alt="logo" />
+      <h1>
+        Eric F. Olsen
+      </h1>
+      <h3>
+        Software Engineer
+      </h3>
```

Comment:
For accessibility, you are not supposed to skip heading levels.  In this case, it doesn't look like it's actually a header, so you could just make it a <div> and pass in attributes to make it look how you want it to look.  If you want to get into a 3rd party UI, with fanciar buttons than the vanilla buttons and text, etc., you may want to look into using Material UI (MUI).

Reply:
@jesseolsen That's good to know! I'm reading-up on that now, but if you have a minute the HTML was taken in your comment so it didn't display properly here, would you please retry with it in a code block?

Reply:
@jesseolsen from what I've read, I'm guessing you suggested to use `<p class="subtitle">Software Engineer</p>` or something to that effect, then style it to be the size I want?

Reply:
I made the change from h3 to p with a css class. Please see the updated BEFORE and AFTER screenshots. It does change how "Software Engineer" looks, but I like it not bold, and played with the size to where I think it looks good. Thoughts?